### ISSUE TITLE
2025-11-20: vllm-project/vllm 仓库更新报告

### ISSUE THREAD

**Issue by yan97ao** [yan97ao/argus#439]
# 每日更新报告（2025-11-20）

## vllm-project/vllm

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-11-20 20:14:29 | Fanli Lin | [BugFix] Fix flash_attn import in `siglip2navit.py` (#29082) |
| 2025-11-20 19:05:50 | Zhewen Li | [Bugfix] Fix spec decode memory regression after #28549 (#28819) |
| 2025-11-20 19:01:03 | Samit | [RL] Add Pause and Resume Generation for Asynchronous RL Training (#28037) |
| 2025-11-20 19:00:19 | Shinichi Hemmi | [MODEL] Implement plamo3 (#28834) |
| 2025-11-20 18:55:10 | Or Ozeri | [BugFix] kv_offloading: Fix bug in loading of partial cpu blocks (#28951) |
| 2025-11-20 18:54:01 | Pleaplusone | [ROCm] Add AMD GPU support on Deepseek v3.2 and SparseMLA (#26670) |
| 2025-11-20 18:53:50 | Anna Shors | Add truncate arg to yarn to match openai implementation of gpt-oss (#28244) |
| 2025-11-20 18:53:09 | cjackal | [Chore] Update `xgrammar` version from 0.1.25 to 0.1.27 (#28221) |
| 2025-11-20 18:52:53 | Jinzhen Lin | [Misc] don't cache `CUTLASS_REVISION` var in CMakeLists.txt (#28518) |
| 2025-11-20 18:52:36 | Dezhan | [BugFix] Fix Llama4 Pipeline Parallelism Assert Error (#28577) |
| 2025-11-20 18:52:02 | Vensen | [Bugfix][cache_kernels]: Fix OOB in cache_kernels.cu (#28760) |
| 2025-11-20 18:51:36 | Boyuan Feng | cleanup at::Tag::needs_fixed_stride_order (#28974) |
| 2025-11-20 17:48:09 | rasmith | [CI/Build][AMD] Fix import errors in tests/kernels/attention (#29032) |
| 2025-11-20 16:52:23 | Wentao Ye | [Bug] Fix torch dynamo warning Dynamo detected a call to a `functools.lru_cache` (#29038) |
| 2025-11-20 15:53:38 | Bradley D | [ci][amd] fix EPLB execution test (#28742) |
| 2025-11-20 15:50:23 | Pleaplusone | [ROCm][BugFix] Fix shared expert loading error when disable `VLLM_ROCM_USE_AITER_FUSION_SHARED_EXPERTS` (#28633) |
| 2025-11-20 14:39:10 | Cyrus Leung | [V0 Deprecation] Remove `num_lookahead_slots` (#29000) |
| 2025-11-20 14:20:12 | Quentin Gallouédec | [Frontend] Allow parsed tool arguments (#28820) |
| 2025-11-20 14:04:23 | Lukas Geiger | [Model][QwenVL] Replace `torch.repeat_interleave` with faster `np.repeat` (#28964) |
| 2025-11-20 13:23:22 | Isotr0py | [Bugfix] Revert custom attention mask for gemma3-mm (#28995) |
| 2025-11-20 12:38:12 | Canlin Guo | [Bugfix] Handle broken frames in video loading (#29001) |
| 2025-11-20 11:50:37 | prashanth058 | [Bugfix] Fix precision loss in LoRA-wrapped RowParallelLinear by fusing bias into GEMM (#28972) |
| 2025-11-20 11:39:36 | Shengliang Xu | Consolidate Nvidia ModelOpt quant config handling for all quantization methods (#28076) |
| 2025-11-20 11:17:48 | Benjamin Chislett | [Feat] Iteration-level profiling for Torch and CUDA profiler (#28987) |
| 2025-11-20 10:37:09 | Fadi Arafeh | [cpu][ci] Add initial set of tests for Arm CPUs (#28657) |
| 2025-11-20 10:11:52 | Qiang Zhang | [AMD] Use Decoupled Kernel Block Size to Support AITER MLA block_size=1 (#27715) |
| 2025-11-20 09:40:49 | Kuntai Du | [DeepSeek + LMCache Multiprocess] handle MLA for deepseek model + LMCache Multiprocess connector (#29039) |
| 2025-11-20 08:39:45 | liangel-02 | [torchao] fix safetensors for sharding (#28169) |
| 2025-11-20 08:26:04 | Nick Hill | [Misc] Colorize logs (#29017) |
| 2025-11-20 08:10:22 | Jialin Ouyang | [GC Debugger] Simply and improve GC Debugger Utils (#29029) |
| 2025-11-20 07:53:15 | Wentao Ye | [Refactor] Optimize `select_experts` (#28069) |
| 2025-11-20 07:47:13 | Alexander Matveev | [Performance] Reduce DeepGEMM N dim restriction from 128 to 64 multiplier  (#28687) |
| 2025-11-20 07:30:08 | JartX | [Feature] EPLB on Qwen3VLMoe and CompressedTensorsWNA16MoEMethod (#28849) |
| 2025-11-20 06:22:43 | Wentao Ye | [CI] Fix precommit `rope_theta` issue (#29040) |
| 2025-11-20 05:54:15 | Max Hu | [Bugfix] Move flashinfer kernel check into ```__init__``` function of ```FusedMoE``` (#29018) |
| 2025-11-20 05:43:54 | Lucas Wilkinson | [BugFix] Fix false assertion with spec-decode=[2,4,..] and TP>2 (#29036) |
| 2025-11-20 05:31:33 | Micah Williamson | [ROCm][CI] Fix Weight Loading With Multiple GPU Tests on ROCm (#28984) |
| 2025-11-20 05:30:04 | Yongye Zhu | [DeepSeek] Fix DeepSeek V3.2 Rope Embedding (#28968) |
| 2025-11-20 05:29:06 | Shu Wang | [MoE] Nvfp4 Masked Gemm: Add flashinfer grouped_gemm_nt_masked (#25990) |
| 2025-11-20 05:20:58 | Julien Denize | [BugFix] Ray with multiple nodes (#28873) |
| 2025-11-20 05:18:32 | Wentao Ye | [Bug] Fix Batch Invariant MLA test (#28967) |
| 2025-11-20 05:04:36 | Ryan Rock | [CI/Build] Fix test_prefix_prefill for AMD (#28905) |
| 2025-11-20 04:52:44 | Qiu | [Feature] Prefill Context Parallel (PCP) basic support (#28718) |
| 2025-11-20 04:01:05 | Izzy Putterman | Eagle: MM Cuda Graphs with MRope (#28896) |
| 2025-11-20 03:59:30 | Aleksandr Malyshev | Upstreaming aiter triton attention backend as a new backend (#28701) |
| 2025-11-20 03:30:57 | 杰兮 | [Bugfix]  Fix precision corruption when shared_experts_stream=None (#28942) |
| 2025-11-20 03:23:54 | Robert Shaw | [Kernels] Improve H200 Fused MoE Config (#28992) |
| 2025-11-20 03:20:22 | Jialin Ouyang | [Core] Reuse created spec tokens lists to mitigate GC cost (#28917) |
| 2025-11-20 02:32:19 | Copilot | Re-enable FlashInfer for Llama4 on Blackwell in e2e fusion tests (#28966) |
| 2025-11-20 01:32:55 | Yuxuan Zhang | GLM-V video segmentation solution adjustment (#28941) |
| 2025-11-20 01:06:36 | Harry Mellor | Update `rope_scaling` to `rope_parameters` in preparation for Transformers v5 (#28542) |
| 2025-11-20 00:24:55 | Shanshan Shen | [Model][Mamba] Add selector for mamba attention backend and make it pluggable for other device (#26487) |
